pinot/app/cli.py

```diff
@@ -75,20 +120,33 @@ def run(args):
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
-    parser.add_argument('--layer', default='GraphConv', type=str)
-    parser.add_argument('--noise_model', default='normal-heteroschedastic', type=str)
-    parser.add_argument('--optimizer', default='adam', type=str)
-    parser.add_argument(
-        '--config',
-        nargs='*',
-        default=[32, 'tanh', 32, 'tanh', 32, 'tanh'])
-    parser.add_argument('--out', default='result', type=str)
-    parser.add_argument('--data', default='esol')
-    parser.add_argument('--batch_size', default=32, type=int)
-    parser.add_argument('--opt', default='Adam')
-    parser.add_argument('--lr', default=1e-5, type=float)
-    parser.add_argument('--partition', default='4:1', type=str)
-    parser.add_argument('--n_epochs', default=5)
+    # With pretrained generative model
+    pretrained_gen_group = parser.add_argument_group("With pretrained generative model:")
+    pretrained_gen_group.add_argument('--pretrained_gen_model', default=None, type=str, help="Parameter file of pretrained generative model")
+
+    # With no pretrained generative model
+    group = parser.add_argument_group("With no pretrained generative model")
+    group.add_argument('--layer', type=str, default='GraphConv', help="Type of graph convolution layer")
+    group.add_argument('--hidden_dims_gvae',nargs='+', type=int, default=[128, 128], help="Hidden dimensions of the convolution layers")
```

Comment:
shall we keep the door open for sneak in activations and dropout for this? in the same fashion as Net

Reply:
yeah, we certainly can